cmd/protoc-gen-go-vtproto/main.go

```diff
@@ -22,8 +22,10 @@ func main() {
 	var f flag.FlagSet
 
 	f.BoolVar(&cfg.AllowEmpty, "allow-empty", false, "allow generation of empty files")
-	cfg.Poolable = make(generator.ObjectSet)
+	cfg.Poolable = generator.NewObjectSet()
+	cfg.PoolableExclude = generator.NewObjectSet()
 	f.Var(&cfg.Poolable, "pool", "use memory pooling for this object")
+	f.Var(&cfg.PoolableExclude, "pool-exclude", "do not use memory pooling for this object")
```

Comment:
Is this a feature we really need? I'd rather drop this flag. It should always be able to write a more careful pattern for `Poolable` to make sure it doesn't match any unnecessary structs.